### PR TITLE
CosmosDB Preview changes for 2022-11-15-preview specification.

### PR DESCRIPTION
**Pull request by rsharm7**
## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/CosmosDB/CosmosDB.Test/CosmosDB.Test.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.1" />
     <PackageReference Include="Microsoft.Azure.Management.Network" Version="26.0.0" />
-    <PackageReference Include="Microsoft.Azure.Management.CosmosDB" Version="3.9.1-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.CosmosDB" Version="3.9.2-preview" />
```
Comment (rsharm7) [inline src/CosmosDB/CosmosDB.Test/CosmosDB.Test.csproj:9]: Release dotnet sdk

File: src/CosmosDB/CosmosDB/ChangeLog.md [hunk 2]
```diff
@@ -25,6 +25,10 @@
     - Added GremlinDatabasesToRestore and TablesToRestore field to provision and restore database account api.
     - Added StartTime and EndTime support for listing restorable containers event feed.
 
+## Version 1.9.2
```
Comment (rsharm7) [inline src/CosmosDB/CosmosDB/ChangeLog.md:28]: Check for version

**Comment by wyunchi-ms** [comment]
The build fails, please fix it.

File: src/CosmosDB/CosmosDB.Test/CosmosDB.Test.csproj (hunk 1 above)
Comment (rsharm7) [reply src/CosmosDB/CosmosDB.Test/CosmosDB.Test.csproj]: not required

**Comment by wyunchi-ms** [comment]
I think Dinesh are also upgrading SDK. Please wait him finish and sync with his code. https://github.com/Azure/azure-powershell/pull/21495

**Pull request closed without merging**